### DISCUSSION TITLE
Can zellij bind a key to run a command in a float panel

### DISCUSSION THREAD

**Discussion by lisiur** [zellij-org/zellij#2219] [Q&A]
In the example from the documentation:

```
bind "a" {
        Run "tail" "-f" "foo" {
            cwd "/tmp"
            direction "Down"
        }
    }
```
I learned that Zellij allows you to bind a key to run a command in a new panel. However, can the direction of the new panel be either floating or something else?

**Comment by arnm** [comment]
Related ask, I'd like to bind a key to run in the current pane
